crypto/x509v3/v3_addr.c

```diff
@@ -1216,7 +1229,11 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
      */
     for (i++; i < sk_X509_num(chain); i++) {
         x = sk_X509_value(chain, i);
-        OPENSSL_assert(x != NULL);
+        if (!ossl_assert(x != NULL)) {
```

Comment:
I'm with Rich on this one, i.e. I'd remove assert. Loop termination condition is effectively equivalent to this check. In other words NULL is not allowed/recognized value.